tests/aframe.test.mjs

```diff
@@ -85,3 +85,34 @@ test.describe('Navigation', () => {
     expect(title).not.toBe('A-Frame Test');
   });
 });
+
+test.describe('Prebid', () => {
+  async function injectIFrame(page, url, image) {
+    await page.waitForFunction(() => document.querySelector('#zesty-div-medium-rectangle') != null);
+    await page.evaluate(([url, image]) => {
+      const iframe = document.createElement('iframe');
+      iframe.id = 'injected';
+      document.querySelector('#zesty-div-medium-rectangle').appendChild(iframe)
+      iframe.contentDocument.write(`<html><body><a href="${url}"><img src="${image}"></a></body></html>`);
+    }, [url, image]);
+  }
+
+  test('Ad creative is loaded once bids is no longer null', async ({ page }) => {
+    const banner = await page.locator('#banner1 > a-plane');
+    await injectIFrame(page, 'https://www.example.com', 'https://picsum.photos/300/250');
```

Comment:
Do you think if we're repeating the url/image pairing again and again, they could be a shared constant across the test files?

Reply:
Good point yeah, I'll move them to some constants.

Reply:
Thanks 👍🏼